catroid/src/androidTest/java/org/catrobat/catroid/uiespresso/formulaeditor/utils/UserDataItemRVInteractionWrapper.java

```diff
@@ -68,6 +68,17 @@ public void performRename(String newName) {
 				.perform(click());
 	}
 
+	public void performEdit(String newValue) {
+		onChildView(R.id.title_view)
+				.perform(longClick());
+		onView(withText(R.string.edit))
+				.perform(click());
+		onView(withId(R.id.input_edit_text))
+				.perform(replaceText(newValue), closeSoftKeyboard());
+		onView(withText(R.string.ok))
```

Comment:
Please change R.string.ok to R.string.save, as changed in the Edit Dialog.

Reply:
Oh sorry, doing it.